pkg/catalog/catalog.go

```diff
@@ -671,7 +671,7 @@ func (c *Catalog) CreateEntry(ctx context.Context, repository string, branch str
 	if err != nil {
 		return err
 	}
-	return c.Store.Set(ctx, repositoryID, branchID, key, *value)
+	return c.Store.Set(ctx, repositoryID, branchID, key, *value, writeConditions...)
```

Comment:
Style nit: no real reason to have varargs here, this API only ever gets generated slices (here on the server side).  varargs now means we cannot varargs later, when we might really want it.